### PR TITLE
finished databases_week3 homework.

### PR DESCRIPTION
**Pull request by shravanipagilla**

### CODE REVIEW

File: databases/week3/create_queries.sql [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+-- 2.Get meals that still has available reservations
+SELECT
+    Meal.title,
+    Meal.max_reservations, (
+        Meal.max_reservations - sum(Reservation.number_of_guests)
+    ) AS available_reservations
+FROM Meal
+    JOIN Reservation ON Meal.id = Reservation.meal_id
+GROUP BY Meal.id;
```
Comment (Elia-Git) [inline databases/week3/create_queries.sql]: I would modify this as below instead.
```
SELECT
    Meal.title,
    Meal.max_reservations, sum(Reservation.number_of_guests) AS booked spots
FROM Meal
    JOIN Reservation ON meal.id = Reservation.meal_id
GROUP BY Reservation.meal_id
HAVING Meal.max_reservations> booked spots;
```

File: databases/week3/create_queries.sql [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+Join Review on Review.meal_id = Meal.id  
+WHERE Review.description = 'good';
+
+-- 7.Get reservations for a specific meal sorted by created_date
+
+/*SELECT Meal.title,Meal.price
+FROM Meal
+INNER JOIN Reservation
+ON Meal.id = Reservation.meal_id
+ORDER BY `Reservation`.`created_date`;*/
```
Comment (Elia-Git) [inline databases/week3/create_queries.sql]: Don't commit commented or empty lines of code!

File: databases/week3/create_queries.sql [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+--5.Get only specific number of meals fx return only 5 meals
+SELECT *  
+FROM Meal LIMIT 5;
+
+-- 6.Get the meals that have good reviews
+SELECT *
+FROM Meal
+Join Review on Review.meal_id = Meal.id  
+WHERE Review.description = 'good';
```
Comment (Elia-Git) [inline databases/week3/create_queries.sql]: Here all meal descriptions might not have qualitative description. Instead we should classify meals into bad, moderate or good as per the nr reviews (stars given).
Try like this..
```
SELECT Meal.*,review.stars AS Stars_recieved FROM Meal
JOIN Review ON meal.id=Review.meal_Id
WHERE Review.stars>20;
```

File: databases/week3/create_queries.sql [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+FROM Meal
+INNER JOIN Reservation
+ON Meal.id = Reservation.meal_id
+ORDER BY `Reservation`.`created_date`;*/
+
+SELECT Meal.title,Meal.price
+FROM Meal
+    INNER JOIN Reservation ON Reservation.meal_id = Meal.id
+WHERE Meal.title = 'burger'
+ORDER BY `Reservation`.`created_date`;
```
Comment (Elia-Git) [inline databases/week3/create_queries.sql]: Why parenthesis around `Reservation.created_date` ...? make sure if the query can run with parenthesis. 
Pls try like this
```
SELECT Meal.title,Meal.price
FROM Meal
    INNER JOIN Reservation ON Reservation.meal_id = Meal.id
WHERE Meal_id = 2
ORDER BY Reservation.created_date ASC;
```

File: databases/week3/create_queries.sql [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+-- 8.Sort all meals by average number of stars in the reviews
+
+SELECT AVG(Review.stars)  AS average
+FROM Meal
+INNER JOIN Review ON Meal.id = Review.meal_id
+GROUP BY Review.meal_id
+ORDER BY AVG(Review.stars);
```
Comment (Elia-Git) [inline databases/week3/create_queries.sql:115]: Good here.

File: databases/week3/create_queries.sql [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+SET  price= 90, title = 'burger'
+WHERE id = 2;
+
+-- 5.Delete a meal with any id, fx 1
+delete from Meal 
+where id = 4;
+
+
+
+                           -- Reservation queries..
```
Comment (Elia-Git) [inline databases/week3/create_queries.sql]: Remove empty lines,not a good practice

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Pls consider to implement given feedback-

**Comment by shravanipagilla** [comment]
@Elia-Git  Thank you very much for your comments and review. I resolved all the comments.

**Pull request merged**